docker-run.sh

```diff
@@ -26,17 +30,26 @@ OUTPUT_CODED_DIR=${12}
 OUTPUT_MESSAGES_CSV=${13}
 OUTPUT_INDIVIDUALS_CSV=${14}
 OUTPUT_PRODUCTION_CSV=${15}
+DRIVE_AUTH_FILE=${16}
```

Comment:
I think I would prefer AVF_SERVICE_ACCOUNT_CREDENTIALS_FILE

Partially to make it explicit what kind of mechanism we're using to auth to drive and partially to make it clear that this credential can do a lot more than 'just' drive access